seqr/views/apis/anvil_workspace_api.py

```diff
@@ -79,6 +80,11 @@ def create_project_from_workspace(request, namespace, name):
     # Add the seqr service account to the corresponding AnVIL workspace
     add_service_account(request.user, namespace, name)
 
+    # Validate the data path
+    if not request_json['dataPath'].startswith("gs://") or not does_file_exist(request_json['dataPath']):
```

Comment:
So all of the files for a given workspace are in one google bucket, the name of which is shown on the main workspace page. But when you actually look at the files in the workspace, it just shows you the relative paths. I think expecting users to go and find the root google bucket name is a bit too confusing, and also not necessary as we should be able to programmatically get the bucket associated with the workspace. Can you change this so we get the google bucket from anvil and then only have the users provide the file path, and then validate against the whole thing?

Reply:
Good point. I should be able to get the bucket for the workspace.